legal-api/report-templates/template-parts/common/addresses.html

```diff
@@ -1,6 +1,11 @@
 {% if business.legalType not in ['GP', 'SP'] %}
 <div class="no-page-break">
-   <div class="separator mt-4"></div>
+   {% if header.name == 'continuationIn' and not business.legalType in ['BEN', 'CBEN', 'CC', 'CCC', 'ULC', 'CUL'] 
+      and not nameRequest.nrNumber %}
+   <!-- Do not render the separator -->
+   {% else %}
+      <div class="separator mt-4"></div>
+   {% endif %}
```

Comment:
Per new design, for numbered company, and without benefit company statement - we will not show the name and statement section. Thus address became the first section, so we need to hide the separator above the address
![image](https://github.com/user-attachments/assets/148ed861-76e2-48e3-ab02-4513cef6afad)